src/helpers/connectors.ts

```diff
@@ -0,0 +1,78 @@
+import { getUrl } from '@/helpers/utils';
+import metamaskIcon from '@/assets/connectors/metamask.png';
+import walletconnectIcon from '@/assets/connectors/walletconnect.png';
+import coinbaseIcon from '@/assets/connectors/coinbase.png';
+import portisIcon from '@/assets/connectors/portis.png';
+import gnosisIcon from '@/assets/connectors/gnosis.png';
+import argentxIcon from '@/assets/connectors/argentx.png';
+
+export default {
+  injected: {
+    id: 'injected',
+    name: 'MetaMask',
+    type: 'injected',
+    root: 'ethereum',
+    icon: metamaskIcon
+  },
+  walletconnect: {
+    id: 'walletconnect',
+    name: 'WalletConnect',
+    network: '1',
+    icon: walletconnectIcon,
+    options: {
+      rpc: {
+        '1': 'https://cloudflare-eth.com',
+        '4': 'https://eth-rinkeby.alchemyapi.io/v2/twReQE9Px03E-E_N_Fbb3OVF7YgHxoGq',
+        '42': 'https://eth-kovan.alchemyapi.io/v2/QCsM2iU0bQ49eGDmZ7-Y--Wpu0lVWXSO'
+      }
+    }
+  },
+  walletlink: {
+    id: 'walletlink',
+    name: 'Coinbase',
+    network: '1',
+    icon: coinbaseIcon,
+    options: {
+      appName: 'Snapshot',
+      darkMode: false,
+      chainId: 1,
+      ethJsonrpcUrl: 'https://cloudflare-eth.com'
+    }
+  },
+  portis: {
+    id: 'portis',
+    name: 'Portis',
+    network: '1',
+    icon: portisIcon,
+    options: {
+      dappId: '3eb93706-c71d-456b-b4eb-322ea27f7d48',
+      network: 'mainnet'
+    }
+  },
+  gnosis: {
+    id: 'gnosis',
+    type: 'gnosis',
+    name: 'Gnosis Safe',
+    icon: gnosisIcon
+  },
+  argentx: {
+    id: 'argentx',
+    name: 'Argent X',
+    type: 'injected',
+    root: 'starknet',
+    icon: argentxIcon
+  }
+};
+
+export function mapConnectorId(sourceName: string) {
+  if (sourceName === 'metamask') return 'injected';
+  if (sourceName === 'coinbase') return 'walletlink';
+
+  return sourceName;
+}
+
+export function getConnectorIconUrl(url) {
```

Comment:
We could add our own function to replace getUrl from helper to transform urls like `ipfs://` with the defined `VITE_IPFS_GATEWAY` and also support `https://` urls.
https://github.com/snapshot-labs/sx-ui/blob/a64a801126173baf069dfe34f5ba153186f38869/src/helpers/utils.ts#L34-L36